contracts/ZkLink.sol

```diff
@@ -528,6 +522,22 @@ contract ZkLink is ReentrancyGuard, Storage, Events, UpgradeableMaster {
             ));
     }
 
+    /// @notice Checks that withdraw params is correct
+    function verifyWithdraw(uint16 tokenId, uint128 amount, uint8 withdrawToL1) internal view {
```

Comment:
checkWithdraw will be better

Reply:
name style keep same with verifyChangePubkey